Source/DafnyDriver/Commands/TranslateCommand.cs

```diff
@@ -5,37 +5,42 @@
 
 namespace Microsoft.Dafny;
 
-class TranslateCommand : ICommandSpec {
-  public IEnumerable<Option> Options =>
+static class TranslateCommand {
+  public static IEnumerable<Option> Options =>
     new Option[] {
       CommonOptionBag.Output,
       CommonOptionBag.IncludeRuntimeOption,
-    }.Concat(ICommandSpec.TranslationOptions).
-      Concat(ICommandSpec.ConsoleOutputOptions).
-      Concat(ICommandSpec.ResolverOptions);
-
-  private static readonly Argument<string> Target = new("language", @"
-cs - Translate to C#.
-go - Translate to Go.
-js - Translate to JavaScript.
-java - Translate to Java.
-py - Translate to Python.
-cpp - Translate to C++.
-
-Note that the C++ backend has various limitations (see Docs/Compilation/Cpp.md). This includes lack of support for BigIntegers (aka int), most higher order functions, and advanced features like traits or co-inductive types.".TrimStart()
-  );
-
-  public Command Create() {
+    }.Concat(DafnyCommands.TranslationOptions).
+      Concat(DafnyCommands.ConsoleOutputOptions).
+      Concat(DafnyCommands.ResolverOptions);
+
+  public static Command Create() {
     var result = new Command("translate", "Translate Dafny sources to source and build files in a specified language.");
-    result.AddArgument(Target);
-    result.AddArgument(ICommandSpec.FilesArgument);
-    return result;
-  }
 
-  public void PostProcess(DafnyOptions dafnyOptions, Options options, InvocationContext context) {
-    dafnyOptions.Compile = false;
-    var noVerify = dafnyOptions.Get(BoogieOptionBag.NoVerify);
-    dafnyOptions.CompilerName = context.ParseResult.GetValueForArgument(Target);
-    dafnyOptions.SpillTargetCode = noVerify ? 3U : 2U;
+    result.AddCommand(new Command("cs", "C#"));
+    result.AddCommand(new Command("go", "GoLang"));
+    result.AddCommand(new Command("js", "JavaScript"));
+    result.AddCommand(new Command("java"));
+    result.AddCommand(new Command("py", "Python"));
+    result.AddCommand(new Command("cpp", "C++. This back-end has various limitations (see Docs/Compilation/Cpp.md). This includes lack of support for BigIntegers (aka int), most higher order functions, and advanced features like traits or co-inductive types."));
```

Comment:
This is probably the best we can do and I like the change to subcommands too much to complain about it. But do we have any integration tests that assert the output of `--help`?

Reply:
What is there to complain about?

> But do we have any integration tests that assert the output of --help?

I don't think so. Would be a type of regression test, breaking every time the options are changed.

Reply:
> What is there to complain about?

It looked like you were cramming the disclaimer into a "full language name" property. But after pulling down your branch and exploring the `Command` types I see it's a "description" property, and the real shortcoming is the fact that the other subcommands aren't providing an actual description. And that leads to this:

```
dafny translate cs --help
Description:
  C#

Usage:
  Dafny translate cs [<file>...] [options]
...
```

So we should really make the descriptions things like "Translate Dafny sources to C# source and build files."

Reply:
On a similar note what's up with the `[]`s?

```
Commands:
  cs <file>    C# []
...
```

Reply:
Fixed both issues